templates/Default/engine/Default/includes/SectorPlayers.inc

```diff
@@ -1,3 +1,18 @@
+<?php
+function getPlayerOptionClass(&$player, &$other) {
```

Comment:
Could you rename this to something like `getDiplomaticStatus` or something and add it to `SmrPlayer` as eg `$player->getDiplomaticStatus($otherPlayer)`, as it's something that would make sense to live there, and gets this logic out of the template